### PR TITLE
Test update node and npm

### PR DESCRIPTION
**Pull request by alex-bezek**
### Summary
Testing updating to node v8 and npm v5.

### Additional Details
Tested out the 1 component on the site and its still functional. Freshly installed node modules and everything

Thanks for contributing to Terra. 
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -47,11 +47,12 @@ lib-cov
...
 pids
 reports
 screenshots
 target
 **/aggregated-translations/*.*
 lib
 props-table
-package-lock.json
\ No newline at end of file
+package-lock.json
```
Comment (KevinJJackson) [inline .gitignore]: package-lock specified twice?

Comment (alex-bezek) [inline .gitignore]: oh, didn't even see it was specified before. Could have sworn it kept coming up in the `git status` will remove. Thanks

**Comment by KevinJJackson** [comment]
Should this change effect the root package.json? https://github.com/cerner/terra-consumer/blob/master/package.json#L7

**Comment by emilyrohrbough** [comment]
Do you need to [here](https://github.com/cerner/terra-consumer/blob/master/package.json#L7) in the package.json as well?

**Comment by alex-bezek** [comment]
I'm assuming so, was waiting to see the response over here https://github.com/cerner/terra-core/pull/722 rather than having the conversation twice

File: .nvmrc [hunk 2]
```diff
@@ -1 +1 @@
-6.11.1
+8.3.0
```
Comment (bjankord) [inline .nvmrc]: We bumped this to 8.4.0 in terra-core.

**Comment by bjankord** [comment]
@alex-bezek We updated the node engine in the root package.json as well in this PR: https://github.com/cerner/terra-core/pull/722/files

**Comment by alex-bezek** [comment]
@bjankord I have updated this PR

**Pull request merged**